docs/user-guide/admin-cli.md

```diff
@@ -653,17 +698,112 @@ enhancement tracked [here](https://github.com/vmware/docker-volume-vsphere/issue
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy rm --name=some-policy
 Successfully removed policy: some-policy
 ```
+
+## Config
+
+Creates, removes, moves and reports on status of config DB. Config DB keep authentication information, and without initializing it no access control is supported. Also, configuring access control will fail, e.g.
+```
+[root@localhost:~] vmdkops_admin  vm-group create --name MY
+Internal Error(Error: Please init configuration in vmdkops_admin before trying to change it)
+```
+
+#### Init
+
+Initialized the config is optional. If the config is not initialized, there will be no access control and all `vm-group` commands will fail with appropriate messages.
+
+Before configuring access control or quotas, the config needs to be inited to either local (`init --local`) or shared (`init --datastore=ds_name) mode
+
+```
+[root@localhost:~] vmdkops_admin config init -h
+usage: vmdkops_admin.py config init [-h] [--local] [--force]
+                                    [--datastore DATASTORE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --local               Allows local (SingleNode) Init
+  --force               Force operation, ignore warnings
+  --datastore DATASTORE
+                        Config DB will be placed on a shared datastore
+```
+
+Example:
+
+```
+[root@localhost:~] vmdkops_admin status
+Version: 0.12.fea683a-0.0.1
+Status: Running
+DB_LocalPath: /etc/vmware/vmdkops/auth-db
```

Comment:
Ok, this is the config DB status, can it be supported as "vmdkops_admin config status"?